### DISCUSSION TITLE
Prometheus Latest Version

### DISCUSSION THREAD

**Discussion by kasarlanagaraju** [prometheus/prometheus#13146] [Q&A]
Hello everyone, we are eagerly anticipating the release of Prometheus's latest version, V2.48.0, as we are currently grappling with numerous vulnerabilities in our project. We believe that updating to the latest version will help us address these issues and enhance the security of our system.

May I know when we can execpet the Latest release?

**Comment by lepichon** [comment]
just been released today https://quay.io/repository/prometheus/prometheus?tab=history
